language-plutus-core/test/evaluation/verifySignature.plc

```diff
@@ -0,0 +1,5 @@
+(program 0.1.0
+  -- this test data is from https://ed25519.cr.yp.to/software.html
+  -- the python test data is used here
+  [{(builtin verifySignature) (con 64) (con 32)} (con 64 ! #e5564300c360ac729086e2cc806e828a84877f1eb8e5d974d873e065224901555fb8821590a33bacc61e39701cf9b46bd25bf5f0595bbe24655141438e7a100b) (con 32 ! #d75a980182b10ab7d54bfed3c964073a0ee172f3daa62325af021a68f707511ad75a980182b10ab7d54bfed3c964073a0ee172f3daa62325af021a68f707511a) (con 0 ! #) ]
```

Comment:
I'd much prefer to provide our own functions to create this. Note that we'll need that for the mockchain tests, since they need to produce signatures for us to verify!

Reply:
I think that would be a separate PR, no? I know the upstream library has a `sign` feature.

I think using test data from somewhere else is at least *sufficient* to know ours is correct.

Reply:
If you like. I'd probably just do it here since it also improves the testing because we can generate the data from the correct function instead of hardcoding it based on some external data.